src/tlo/analysis/utils.py

```diff
@@ -293,7 +296,10 @@ def generate_series(dataframe: pd.DataFrame) -> pd.Series:
                 df: pd.DataFrame = load_pickled_dataframes(results_folder, draw, run, module)[module][key]
                 output_from_eval: pd.Series = generate_series(df)
                 assert pd.Series == type(output_from_eval), 'Custom command does not generate a pd.Series'
-                res[draw_run] = output_from_eval * get_multiplier(draw, run)
+                if output_from_eval.apply(is_number).all():
```

Comment:
Is this to check that the Series is float or int dtype? Can you use \[is\_numeric\_dtype\](https://pandas.pydata.org/docs/reference/api/pandas.api.types.is_numeric_dtype.html)?

Reply:
Yes, it is. I use the suggested fnc now, thx!

Reply:
Btw, the `is_numeric_dtype` is underlined in pycharm when imported, and if I hold the mouse above it, it says:
`'is_numeric_dtype' is not declared in __all__`

Should I worry about it or it's fine?